src/sync/syncTranslators.js

```diff
@@ -50,6 +50,8 @@ export const RECORD_TYPES = new SyncTranslator({
   Message: 'message',
   Name: 'name',
   NameStoreJoin: 'name_store_join',
+  NameTag: 'name_tag',
+  NameTagJoin: 'name_tag_join',
```

Comment:
Add to: 
translators ✅ 
incoming sync utils case ✅ 
incoming sync utils sanity ✅ 
schema ✅